### PR TITLE
Parameterize HUC 8 codes  projection string

### LINKED ISSUE #4: Parameterize class names
Currently, I've hard coded the names given to the shape classes. We should have a way to easily change these so that we can adjust and name appropriately for different projects.

### PR DESCRIPTION
**Pull request by lindsayplatt**
Round 1 of parameterization. Fixes #4 (parameterizes classes & ids).

### CODE REVIEW

File: 3_build.R [hunk 1]
```diff
@@ -13,38 +13,28 @@ p3_targets <- list(
...
-                    in_svg = g_conus_state_svg,
-                    paths = p2_conus_states_paths,
-                    path_nms = sprintf('state-%s', p2_conus_states_names)),
+    add_poly_group_to_svg(
+      out_svg = "3_build/tmp/state_paths.svg",
+      in_svg = root_svg,
+      grp_id = "conus-states",
+      paths = p2_conus_states_paths,
+      path_ids = p2_conus_states_names,
+      path_class = "state"),
```
Comment (cnell-usgs) [inline 3_build.R]: This class seems to be getting lost. When I run the pipeline I'm seeing `class="NA"` on state paths

File: 3_build.R [hunk 2]
```diff
@@ -13,38 +13,28 @@ p3_targets <- list(
...
-    add_child_paths(out_svg = "3_build/tmp/state_paths.svg",
-                    in_svg = g_conus_state_svg,
-                    paths = p2_conus_states_paths,
-                    path_nms = sprintf('state-%s', p2_conus_states_names)),
+    add_poly_group_to_svg(
+      out_svg = "3_build/tmp/state_paths.svg",
+      in_svg = root_svg,
+      grp_id = "conus-states",
+      paths = p2_conus_states_paths,
+      path_ids = p2_conus_states_names,
```
Comment (cnell-usgs) [inline 3_build.R:23]: need to substitute spaces with `-` or `_` in class and ids

File: 3_build.R (hunk 1 above)
Comment (cnell-usgs) [reply 3_build.R]: The first state has `class="state"` but subsequent paths do not, same for the huc8s

**Review by cnell-usgs — CHANGES_REQUESTED** [review]
Just need to apply the classes to all elements, but otherwise looks good

File: 3_build.R (hunk 1 above)
Comment (lindsayplatt) [reply 3_build.R]: The trick was that I needed to repeat the values passed in here!

**Pull request merged**